tests/restart/52-cycle-point-time-zone.t

```diff
@@ -46,20 +46,24 @@ run_ok "${TEST_NAME_BASE}-validate" cylc validate "${SUITE_NAME}"
 # Set time zone to +01:00
 export TZ=BST-1
 
-suite_run_ok "${TEST_NAME_BASE}-run" cylc run "${SUITE_NAME}"
+suite_run_ok "${TEST_NAME_BASE}-run" cylc run "${SUITE_NAME}" --hold
+poll_suite_running
 dumpdbtables
 cmp_ok 'dump.out' <<< 'cycle_point_tz|+0100'
 
 cylc stop "${SUITE_NAME}"
+poll_suite_stopped
 
 # Simulate DST change
 export TZ=UTC
 
-suite_run_ok "${TEST_NAME_BASE}-restart" cylc restart "${SUITE_NAME}"
+suite_run_ok "${TEST_NAME_BASE}-restart" cylc restart "${SUITE_NAME}" --hold
```

Comment:
When running suites, unless you want the workflow itself to progress it's best to start in "held" mode so that nothing in the workflow progresses.